plugins/inventory/flightctl.py

```
@@ -268,6 +273,109 @@ def _create_tmp_crt(self, encoded_data):
             # Ensure the created temp file is deleted when our module exits
             self.add_cleanup_file(temp_file.name)
 
+    def _fetch_auth_config(self, host: str, verify_ssl: bool,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)